internal/avs/external_eval_assistant.go

```diff
@@ -48,8 +49,51 @@ func (eea *ExternalEvalAssistant) ProvideParentId(_ internal.ProvisioningParamet
 	return eea.avsConfig.ParentId
 }
 
-func (eea *ExternalEvalAssistant) ProvideTags() []*Tag {
-	return eea.avsConfig.ExternalTesterTags
+func (eea *ExternalEvalAssistant) ProvideTags(pp internal.ProvisioningParameters) []*Tag {
+	var Tags []*Tag
+	Tags = append(Tags, &Tag{
+		Content:      pp.ErsContext.TenantID,
```

Comment:
InstanceID field should be used here.

Reply:
i cant find instanceID inside Provisioning Parameters, or its called differently... :(